install.sh

```diff
@@ -96,7 +99,7 @@ function extract_install () {
     rm -rf ./usr
     tar -xf $2
     echo "Installing $1 (this may take a while)..."
-    sudo tar cf - ./usr/* | (cd /; sudo tar xp --keep-directory-symlink -f -)
+    tar cf - ./usr/*
```

Comment:
Do we need this?  From what I can tell, it serves no purpose and will never execute anyway.